eng/common/scripts/apiview-helper.ps1

```diff
@@ -0,0 +1,56 @@
+
+
+function Map-LanguageToAPiview-Filter($language)
+{
+    $lang = $language
+    # Update language name to match those in API cosmos DB. Cosmos SQL is case sensitive and handling this within the query makes it slow.
+    if($lang -eq 'javascript'){
+        $lang = "JavaScript"
+    }
+    elseif ($lang -eq "dotnet"){
+        $lang = "C#"
+    }
+    elseif ($lang -eq "java"){
+        $lang = "Java"
+    }
+    elseif ($lang -eq "python"){
+        $lang = "Python"
+    }
+    return $lang
+}
+
+function Check-ApiReviewStatus($packageName, $language)
+{
+  # Get API view URL and API Key to check status
+  Login-AzAccount
+  $url = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIURL -AsPlainText
```

Comment:
Who has access to this KV?

We should make sure to add some error handling for folks that don't have permissions so they have some idea of what to do to fix the issue.

Reply:
ADP members and partner DL as per our earlier discussions. I have added point in APIView wiki FAQ